arm-trafficmanager/2017-05-01/swagger/trafficmanager.json

```diff
@@ -0,0 +1,972 @@
+               "$ref": "#/definitions/DeleteOperationResult" 
+             } 
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Profiles"
+        ],
+        "operationId": "Profiles_Update",
+        "description": "Update a Traffic Manager profile.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group containing the Traffic Manager profile."
+          },
+          {
+            "name": "profileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager profile."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Profile"
+            },
+            "description": "The Traffic Manager profile parameters supplied to the Update operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": "The updated Traffic Manager profile.", 
+             "schema": { 
+               "$ref": "#/definitions/Profile" 
+             } 
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Network/trafficManagerGeographicHierarchies/default": {
+      "get": {
+        "tags": [
+          "GeographicHierarchies"
+        ],
+        "operationId": "GeographicHierarchies_GetDefault",
+        "description": "Gets the default Geographic Hierarchy used by the Geographic traffic routing method.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": "The default Geographic Hierarchy.", 
+             "schema": { 
+               "$ref": "#/definitions/TrafficManagerGeographicHierarchy" 
+             } 
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
```

Comment:
There are many `description` elements missing. Please refer the CI to fix and help us make the great documentation from this API :)

Example: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/227979765#L1499 Search for word `DescriptionMissing `